app/controllers/system_admin/providers_controller.rb

```diff
@@ -25,6 +25,19 @@ def show
       @users_view = UsersView.new(@provider)
     end
 
+    def edit
```

Comment:
Its better to have a before_action here to find the provider. This should be for show, edit, update actions.

Reply:
sorted